pkg/apis/core/validation/shoot.go

```diff
@@ -793,6 +793,10 @@ func ValidateClusterAutoscaler(autoScaler core.ClusterAutoscaler, fldPath *field
 			allErrs = append(allErrs, field.Invalid(fldPath.Child("scaleDownUtilizationThreshold"), *threshold, "can not be greater than 1.0"))
 		}
 	}
+	if maxNodeProvisionTime := autoScaler.MaxNodeProvisionTime; maxNodeProvisionTime != nil && maxNodeProvisionTime.Duration < 0 {
+		allErrs = append(allErrs, field.Invalid(fldPath.Child("maxNodeProvisionTime"), *maxNodeProvisionTime, "can not be negative"))
+	}
+
```

Comment:
It will be nice the add a validation also for the `expander` field.

Reply:
@vpnachev - I have tried to address this with https://github.com/gardener/gardener/pull/4508/commits/26bc7fb22d4c35cf9257518a3b9ccfd0da59a22f. PTAL.

Reply:
Thanks!